### PR TITLE
Closes #1860 print software version all modules

### LINKED ISSUE #1860: Print software versions for all modules
### Description

We only stored software versions for the structural module, but not for the other modules. This is not correct behavior if users repeat their analysis for certain modules only. 

### Tasks

- [x] xASL_qc_CollectParameters: store software versions for all modules 

### How to test

Henk already tested this with the ExploreASL/External/TestDataSet

### Release notes

Store software versions for all modules.

### PR DESCRIPTION
**Pull request by HenkMutsaerts**
### Linked issue

Closes #1860

### CODE REVIEW

File: Functions/QualityControl/xASL_qc_CollectParameters.m [hunk 1]
```diff
@@ -96,18 +96,16 @@
...
-        x.Output.(ScanType).Version_LST = x.Output.SoftwareVersion.LST;
-        
-        % General software versions (put this in structural only, to avoid redundant output
-        x.Output.(ScanType).Version_ExploreASL = x.Output.SoftwareVersion.ExploreASL;
-        x.Output.(ScanType).Version_Matlab = x.Output.SoftwareVersion.Matlab;
-        x.Output.(ScanType).Version_SPM12 = x.Output.SoftwareVersion.SPM12;        
-    case {'dwi', 'func', 'ASL'}
-        x.Output.(ScanType).(x.SESSIONS{iSession}).Version_FSL = x.Output.SoftwareVersion.FSL;
-end
+x.Output.(ScanType).Version_CAT12 = x.Output.SoftwareVersion.CAT12; % CAT12
```
Comment (jan-petr) [inline Functions/QualityControl/xASL_qc_CollectParameters.m]: I agree to print ExploreASL version in all modules. 
But it doesn't IMHO make sense to print CAT12 and LST for other than structural modules.

File: Modules/xASL_module_ASL.m [hunk 2]
```diff
@@ -530,6 +530,19 @@
 end
 
 
+
```
Comment (jan-petr) [inline Modules/xASL_module_ASL.m:533]: what happened here?

**Review by jan-petr — CHANGES_REQUESTED** [review]
Minor questions

File: Modules/xASL_module_ASL.m (hunk 2 above)
Comment (HenkMutsaerts) [reply Modules/xASL_module_ASL.m:533]: It helps for searching through the code if there is more space between the function and the subfunctions, at least for me. See the individual commit messages ;)

File: Functions/QualityControl/xASL_qc_CollectParameters.m (hunk 1 above)
Comment (HenkMutsaerts) [reply Functions/QualityControl/xASL_qc_CollectParameters.m]: Done in 128171247d5b8d8d8a4d73bc7cc72c72846cbc3a

**Review by jan-petr — APPROVED** [review]
OK

**Pull request merged**